<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoNuevaCertificacion</class>
 <widget class="QDialog" name="DialogoNuevaCertificacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>328</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fecha de la nueva certificacion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelDia">
           <property name="text">
            <string>Día:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboDia">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>31</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelMes">
           <property name="text">
            <string>Mes:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboMes">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="labelAnno">
           <property name="text">
            <string>Año:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboAnno">
           <item>
            <property name="text">
             <string>2010</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2011</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2012</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2013</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2014</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2015</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2016</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2017</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2018</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2019</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2020</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2021</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2022</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2023</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2025</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2026</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2027</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2028</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2029</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2030</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2031</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2032</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2033</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2034</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2035</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2036</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2037</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2038</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2039</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2040</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2041</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2042</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2043</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2044</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2045</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2046</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2047</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2048</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2049</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2050</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogoNuevaCertificacion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogoNuevaCertificacion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
